### PR TITLE
RS-213: Rust ReductCLI

### PR DESCRIPTION
**Pull request by atimin**
* Replace Python CLI with Rust implementation
* Write a short update about the migration

### CODE REVIEW

**Comment by atimin** [comment]
@atimin Update landing page
@AnthonyCvn  Check how it work on Apple M platform, check GIF if it possible to do small.

File: src/components/InstallClientCLI/LinuxInstall.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import CodeBlock from '@theme/CodeBlock';
+
+export default function LinuxInstall(props) {
+    const binaryInstall = `
+wget https://github.com/reductstore/reduct-cli/releases/latest/download/reduct-cli.linux-amd64.tar.gz
+tar -xvf reduct-cli.linux-amd64.tar.gz
+chmod +x reduct-cli
+mv reduct-cli /usr/local/bin
```
Comment (AnthonyCvn) [inline src/components/InstallClientCLI/LinuxInstall.tsx:9]: missing `sudo`

**Review by AnthonyCvn — APPROVED** [review]
Nice release note 👍 Only missing a "sudo" on the download page for Linux.

**Comment by AnthonyCvn** [comment]
@atimin check my commit [terminal animation](https://github.com/reductstore/website/pull/30/commits/f00dc3f38158584afa44de98aab485373329390a).

**Comment by atimin** [comment]
@AnthonyCvn Unfortunately, don't work well for the mobile version.

**Comment by AnthonyCvn** [comment]
@atimin should be better

**Pull request merged**